etcdserver/etcdserverpb/rpc.proto

```diff
@@ -78,6 +78,21 @@ message RangeRequest {
   // if the revision has been compacted, ErrCompaction will be returned in
   // response.
   int64 revision = 4;
+
+  enum SortOrder {
+	NONE = 0; // default, no sorting
+	ASCEND = 1; // lowest key first
+	DESCEND = 2; // highest key first
+  }
+  SortOrder sort_order = 5;
+  enum SortAttr {
+	KEY = 0;
```

Comment:
we called this `CompareTarget`. We need to change to `sortTarget` or change `compareTarget` to `CompareAttr`? No strong opinion here.